splash/qtrender_lua.py

```diff
@@ -221,6 +248,10 @@ def init_dispatcher(self, return_func):
         """
         self._return = return_func
 
+    @attr_function(attr_name='js_enabled', default=True)
```

Comment:
It should be a Python property, or do something similar to what are Python properties doing, but in Lua. 

So instead of a single set_js_enabled method you can have one more method, get_js_enabled, call it from Lua and expose the result as an attribute. Instead of these two methods we can also have a Python property, but it'll require some changes to the sandboxing code.